### PR TITLE
GF-341-article-pagination

### PR DESCRIPTION
**Pull request by weited**

### CODE REVIEW

File: src/main/java/com/avengers/gamera/dto/article/MiniArticleGetDto.java [hunk 1]
```diff
@@ -25,7 +20,7 @@ public class MiniArticleGetDto {
     private GameGetDto game;
     private UserGetDto user;
     private String title;
-    private ArticleType type;
+    private EArticleType type;
     private OffsetDateTime createdTime;
     private OffsetDateTime updatedTime;
 }
```
Comment (Additivemenu) [inline src/main/java/com/avengers/gamera/dto/article/MiniArticleGetDto.java:26]: 咱们需要在MiniArticleGetDto里提供跳转到article页面的hyperlink吗? 当用户点击处在news page或review page的article card之后可以跳转到对应页面

Comment (weited) [reply src/main/java/com/avengers/gamera/dto/article/MiniArticleGetDto.java:26]: 这个上面有Article 的id   前端自己拿着id 跳转

**Review by HUHANG716 — COMMENTED** [review]
should order be DESC when getting articles?

**Comment by weited** [comment]
> should order be DESC when getting articles?

yes we can implement this later just by adding sort into the pagination. We have multip different order functions in frontend.

**Pull request merged**